db/re/item_db.conf

```diff
@@ -61939,6 +61940,11 @@ item_db: (
 	Slots: 1
 	Loc: 256
 	View: 525
+	Script: <"
+		bonus bDex, 2;
+		skill TF_DOUBLE, 5;
```

Comment:
I agree with @iRagno , it should not be giving the skill ... 
should be the `bonus bDoubleRate,x;`

since kRO has no way to test it, what about the AEGIS ? does it exist in there? someone take a look there?